azure/image/ubuntu/variables.pkrvars.hcl.example

```diff
@@ -0,0 +1,16 @@
+resource_group        = "${AZURE_RESOURCE_GROUP}"
+subscription_id       = "${AZURE_SUBSCRIPTION_ID}"
+az_gallery_name       = "${GALLERY_NAME}"
+az_gallery_image_name = "${GALLERY_IMAGE_DEF_NAME}"
+podvm_distro          = "${PODVM_DISTRO}"
+
+# If you set this true then don't provide client_id, client_secret & tenant_id.
+# use_azure_cli_auth  = "false"
+client_id             = "${AZURE_CLIENT_ID}"
+client_secret         = "${AZURE_CLIENT_SECRET}"
+tenant_id             = "${AZURE_TENANT_ID}"
+
+# az_gallery_image_version = "0.0.1"
+# az_image_name            = "peer-pod-vmimage"
+# vm_size                  = "Standard_D2as_v5"
```

Comment:
Should we default to a CVM instance size? If yes, this should be `Standard_DC2as_v5` instead.

Reply:
Good point.

Reply:
Oh, I know why this is the case, using packer to do builds using confidential SKU was a problem. Like it errored out saying that it cannot start a VM without adding secure boot or something like that. So during the builds we can still do it on this SKU and use that image to bring up CVMs.

Reply:
Yep done, also some background here: https://github.com/confidential-containers/cloud-api-adaptor/pull/861#discussion_r1183218614